src/MailpitCommands.ts

```diff
@@ -63,12 +63,7 @@ class MailpitCommands {
 				url: this.mailpitUrl(`/v1/messages?start=${start}&limit=${limit}`),
 				auth: this.auth,
 			})
-			.then((response) => {
-				if (response.status !== 200) {
-					throw new Error(`Failed to get messages. Status: ${response.status}, Body: ${response.body}`);
-				}
-				return response.body as MessagesSummary;
-			});
+			.its('body');
```

Comment:
can we not change the return here ?

Reply:
.its('body') is returning the body which is casted to MessageType but if it is wanted I can change it.

https://docs.cypress.io/api/commands/its > Get a property's value on the previously yielded subject.

```suggestion
			.then(response=> response.body as MessagesSummary);
```

I think both working the same.